app/views/smart_answers/custom_result.html.erb

```diff
@@ -0,0 +1,25 @@
+<% # this custom result template is currently tested on the next-steps-for-your-business flow %>
+<% outcome = @presenter.current_node %>
+<% content_for :outcome_title do %>
+  <% if outcome.title.present? %><%= outcome.title %><% else %>Outcome<% end %>
+<% end %>
+<% content_for :head do %>
+  <meta name="robots" content="noindex">
+<% end %>
+
+<% content_for :hero do %>
+  <%= render "components/hero", {
+    title: outcome.title,
+    description: outcome.description,
```

Comment:
I think ultimately we want to move away from ERB templates and use locale files.  (ERB templates create a lot of overhead and I think were originally introduced to support long form govspeak content). 

Instead of adding a new field for the ERB template - we could just use `t("next_steps_for_your_business.hero.description")` which should work out of the box. There would need to be longer term thinking about how we properly organise content - but this would be good enough for now without adding anymore additional code to ERB templates.

Reply:
I thought about that, but then how do we keep this `custom_results` template generic, so that other flows can use it in future? I'm uneasy about coupling this with a specific flow.

Reply:
Yup fair! I guess I was thinking that this would save us having to put in additional functionality into ERB rendering - however, agree we probably don't want to use locales until we have more concrete plan about how we will use them for all flows. Let's stick with your proposed changes for now.